movie-browser/src/view/FeaturedMovie.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import CardDetail from './CardDetail.js'
+
+// TODO: makes toggle for detail view work. have no idea why it's not working for this one
+// toggle: https://stackoverflow.com/questions/29913387/show-hide-components-in-reactjs
+class FeaturedMovie extends React.Component{
+  state = {isDetailShown: false}
+
+  handleClick = () => {
+    this.setState(preState => ({isDetailShown: !preState.isDetailShown}));
+    console.log(this.props.movie)
```

Comment:
remove console.log

Reply:
Removed. I guess it means that I should make pull request an only clean code without unnecessary.